addons/website_slides_sale/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+{
+    'name': "website_slides_sale",
```

Comment:
As name is displayed in the apps kanban view it should have a sexy label :) . Something like "Sell Courses" probably.

Reply:
Oops. Fixed.